src/splitseq_combine.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#Run once for each sample to gather sequences from multiple sequencing runs that belong to that sample.  Sample name must be exactly the same as in each sequencing run.
+
+#USAGE: ./splitseq_combine.sh run1 v1 reference_directory "splitseq_all_out_directories" "sample_name A1:B6"
+
+all_args=("$@")
+
+run_id=$1
+chemistry=$2  #v1 or v2
+ssrefdir=$3
+libdirs="$4"
+sample_wells=("${all_args[@]:4}")
+
+splitseq_venv="/Genomics/grid/users/rleach/local/splitseq"
+splitseq_runs="${splitseq_venv}/splitseq_runs"
+
+cd "${splitseq_venv}"
```

Comment:
I would remove the references to the venv. It would be nice to keep that separate from the scripts. If the user provides the dependencies via venv, great, but they shouldn't be required to, imo anyway.